lldb/source/Plugins/Process/elf-core/RegisterContextPOSIXCore_loongarch64.cpp

```diff
@@ -48,6 +48,10 @@ bool RegisterContextCorePOSIX_loongarch64::ReadGPR() { return true; }
 
 bool RegisterContextCorePOSIX_loongarch64::ReadFPR() { return true; }
 
+bool RegisterContextCorePOSIX_loongarch64::ReadLSX() { return true; }
+
+bool RegisterContextCorePOSIX_loongarch64::ReadLASX() { return true; }
```

Comment:
Do core files work with the code like this?

I thought you were just stubbing out the functions but I see that ReadGPR also does this and I assume that already works. Surprising.

Reply:
I have verified that it works(surprised too). I have removed the ELF-core-related modifications, and support for adding LSX/LASX registers in ELF-core will be included in a subsequent patch. The modifications are too extensive; it's not just about adding two functions.